model/pipeline/src/main/resources/org/apache/beam/model/common_urns.md

```diff
@@ -125,6 +125,8 @@ but are used at runtime and must be supported by every SDK.
 
 ###  beam:coder:windowed_value:v1
 
+###  beam:coder:windowed_value:v2
```

Comment:
@jkff had worked on getting the URNs into an enum. Best to sync with him on that.

Reply:
Ah, good to know, will do when this is out of exploratory stage. With this version I mostly want to clarify the scope, APIs, and unknowns. This code may eventually get thrown away, depending on findings.

Reply:
If it is a [spike](https://en.wikipedia.org/wiki/Spike_(software_development)) in the classic sense, then I do enjoy being able to see the code in a PR, and interested people who don't sit next to Anton can also take a peek.